tools/depends/target/date/Makefile

```diff
@@ -0,0 +1,37 @@
+include ../../Makefile.include
+DEPS= ../../Makefile.include Makefile
+
+LIBNAME=date
+VERSION=2.4.1
+ARCHIVE=v$(VERSION).tar.gz
+BASE_URL := https://github.com/HowardHinnant/$(LIBNAME)/archive
+
+LIBDYLIB := $(PLATFORM)/build/libtz.a
+
+CMAKE_OPTIONS := -DBUILD_SHARED_LIBS=OFF -DENABLE_DATE_TESTING=OFF -DUSE_SYSTEM_TZ_DB=OFF -DUSE_TZ_DB_IN_DOT=OFF
```

Comment:
When using a newer version (3.0.1) this configure will not install date/tz.h header

must add -DBUILD_TZ_LIB=ON for tz.h to be installed

Reply:
I recall I added this to `FindDate.cmake` but I omit this file in Debian at all. Will fix.